atm.js

```diff
@@ -1,21 +1,145 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//Buttons DOM Variables
+var depCheck = $("#depCheck");
```

Comment:
I would move these global variables into another object, as properties on your checkReg/savingReg objects, or on the constructor function's prototype